api/services/events/v1/task.pb.go

```diff
@@ -137,6 +138,184 @@ func init() {
 	proto.RegisterType((*TaskResumed)(nil), "containerd.services.events.v1.TaskResumed")
 	proto.RegisterType((*TaskCheckpointed)(nil), "containerd.services.events.v1.TaskCheckpointed")
 }
+
+// Field returns the value for the given fieldpath as a string, if defined.
+// If the value is not defined, the second value will be false.
+func (m *TaskCreate) Field(fieldpath []string) (string, bool) {
+	if len(fieldpath) == 0 {
+		return "", false
+	}
+
+	switch fieldpath[0] {
+	// unhandled: rootfs
+	// unhandled: pid
```

Comment:
Looks like we don't support integer/floats (and thinking about the Mount type, slices).

`containerd` only has used of `uint32` and `[]string` atm I think.